auth_company_signup/controllers/main.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
to remove

Reply:
no we should keep it as the code is from the Odoo module auth_signup